python/py-graph-tool/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup           compiler_blacklist_versions 1.0
 
 set realname        graph-tool
 name                py-${realname}
-version             2.26
+version             2.27
 revision            3
```

Comment:
The revision needs to be reset to 0 when the version increases. Otherwise, I suspect everything is fine. I am waiting for the installation to finish.